client/src/components/Header.js

```diff
@@ -1,8 +1,16 @@
 import React from "react";
 import AppBar from "@mui/material/AppBar";
-import Container from "@mui/material/Container";
+import styled from "styled-components";
+// import Container from "@mui/material/Container";
 import "../App.css";
 
+const Container = styled.h1`
+ 
+  font-size: 1.5em;
+  text-align: center;
+  }
+`;
```

Comment:
As written, this code places an `h1` element inside another `h1` element.  I assume that wasn't your intent.  My guess is that you still want a MUI `Container` here; perhaps you meant to use the `styled()` factory to _extend_ the Container component styles instead of using a `tagName` alias directly?  Also, FWIW I assume you would prefer to style the `h1` _inside_ the container instead of the container itself?  `styled-components` supports a lightweight version of the SASS selector engine which allows for nesting selectors.  

Putting it all together, I wonder if this is closer to your intent?

```suggestion
const StyledContainer = styled(Container)`
 h1 {
    font-size: 1.5em;
    text-align: center;
  }
`;
```